bin/addonbundle.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+const { join } = require("path");
```

Comment:
Can you use the new version from https://github.com/replugged-org/plugin-template/commit/8d08a035d894ff4cf1b37b59673e3f23dae652fd#diff-56de87ca2ae95ee7c7a031bf487c87288b5dd206b66fed3ab6f551bf58081645

Reply:
It was using the new version lol